plugins/modules/api_info.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: api_info
+
+author:
+  - Tjaž Eržen (@tjazsch)
+
+short_description: Manage ServiceNow GET requests
+description:
+  - Retrieve records via ServiceNow REST Table API for an arbitrary table.
+  - For more information, refer to the ServiceNow REST TAble API documentation at
+    U(https://docs.servicenow.com/bundle/paris-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html#c_TableAPIO).
+version_added: 2.0.0
+extends_documentation_fragment:
+  - servicenow.itsm.instance
+  - servicenow.itsm.sys_id.info
+seealso:
+  - module: servicenow.itsm.api
+options:
+  resource:
+    description:
+      - The name of the table that we want to obtain records from
+    type: str
+    required: true
+  sysparm_query:
+    description:
+      - An encoded query string used to filter the results.
+      - Mutually exclusive with C(sys_id).
```

Comment:
Maybe this no longer applies. Please see the comment below, attached to the code specifying mutual exclusiveness.

Reply:
After code changes regarding mutual exclusiveness with https://github.com/ansible-collections/servicenow.itsm/commit/2a7efdeff2bdf37a136e1953f06cfe479e75323c, updating docs with https://github.com/ansible-collections/servicenow.itsm/pull/179/commits/ba35f6f2c33ed58219e554c731dfbdc8afbc6531.